stock_warehouse_calendar_orderpoint/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module adds an Reordering Calendar to the Warehouse. This calendar
+can then used by reordering rules to compute the forecasted date when goods
```

Comment:
@sebalix You can maybe remind where it can be found ?

https://github.com/odoo/odoo/blob/14.0/addons/stock/static/src/xml/stock_orderpoint.xml#L27